bin/compute_mutdensity.py

```diff
@@ -143,8 +145,17 @@ def load_n_process_inputs(maf_path, depths_path, annot_panel_path, sample_name):
     depths_subset_adj_sample_df = depths_df.merge(annot_panel_df.drop_duplicates(subset = ["CHROM", "POS", "REF", "ALT"])[["CHROM", "POS"]],
                                                     on = ["CHROM", "POS"], how = "inner")
 
-    return maf_df, depths_subset_df, depths_subset_adj_df, depths_subset_adj_sample_df
+    # Add domains and exons to maf_df
+    annot_panel_df['CHROM_POS'] = [f'{chrom}:{pos}' for chrom, pos in zip(annot_panel_df['CHROM'], annot_panel_df['POS'])]
```

Comment:
[Updated] Maybe we could do something like:

```suggestion
    annot_panel_df['CHROM_POS'] = annot_panel_df['CHROM'].astype(str) + ':' + annot_panel_df['POS'].astype(str)
```

The `.astype(str)` is just in case to avoid problems.

Reply:
uhh I did not know that we could use format in this way, I will apply this suggestion